custom_components/google_home/sensor.py

```diff
@@ -95,14 +86,14 @@ def get_device(self):
 class GoogleHomeTokenSensor(GoogleHomeSensorMixin, GoogleHomeTokenEntity):
     """GoogleHome Sensor class."""
 
-    def __init__(self, coordinator, entry, device_name, local_auth_token):
+    def __init__(self, coordinator, entry, name, token):
```

Comment:
Let's keep `device_name` and `auth_token` for arguments. `name` is too generic IMO in this context.

Reply:
But is it? :thinking: 
When it comes to a `device` object of this specific class why to repeat ourselves saying `device_name` and refer to it as `device.device_name` were simply `device.name` is quite elaborate

Reply:
`device.name` is good. But here you don't have `device` anymore, I'm talking only about sensor here.
And I think `auth_token` is a little nicer than just `token` because there're different kinds of tokens which may exist.

Reply:
Yeh, that's reasonable. Thanks, will change :)